content/id/docs/reference/access-authn-authz/rbac.md

```diff
@@ -0,0 +1,1195 @@
+- apiGroups: ["batch", "extensions"]
+  #
+  # pada tingkat HTTP, nama dari sumber daya untuk mengakses objek Job
+  # adalah "jobs"
+  resources: ["jobs"]
+  verbs: ["get", "list", "watch", "create", "update", "patch", "delete"]
+```
+
+Mengizinkan pembacaan ConfigMap bernama "my-config" (harus terikat dengan
+RoleBinding untuk membatasi pada sebuah ConfigMap di sebuah _namespace_):
+
+```yaml
+rules:
+- apiGroups: [""]
+  #
+  # pada tingkat HTTP, nama dari sumber daya untuk mengakses objek ConfigMap
+  # adalah "configmaps"  
+  resources: ["configmaps"]
+  resourceNames: ["my-config"]
+  verbs: ["get"]
+```
+
+Mengizinkan pembacaan sumber daya `"nodes"` pada kumpulan API inti (karena sebuah node
+ada pada lingkup-klaster, ini harus berupa ClusterRole yang terikat dengan ClusterRoleBinding
+agar efektif):
+
+```yaml
+rules:
+- apiGroups: [""]
+  #
+  # pada tingkat HTTP, nama dari sumber daya untuk mengakses objek Node
+  # adalah "nodes"
+  resources: ["nodes"]
+  verbs: ["get", "list", "watch"]
+```
+
+Mengizinkan permintaan GET dan POST kepada _endpoint_ non-sumber daya `/healthz` dan seluruh _subpath_
+(harus berada di dalam ClusterRole yang terikat dengan ClusterRoleBinding agar efektif):
+
+```yaml
+rules:
+- nonResourceURLs: ["/healthz", "/healthz/*"] # '*' in a nonResourceURL is a suffix glob match
+  verbs: ["get", "post"]
+```
+
+### Mengacu Pada Subjek
+
+RoleBinding atau ClusterRoleBinding mengikat sebuah peran ke subjek.
+Subjek dapat berupa kelompok, pengguna atau ServiceAccounts.
+
+Kubernetes merepresentasikan _usernames_ sebagai string.
+Ini bisa berupa: nama sederhana, seperti "alice"; email, seperti "bob@example.com";
+atau ID pengguna numerik yang direpresentasikan sebagai string. Terserah kamu sebagai administrator klaster
+untuk mengkonfigurasi [authentication modules](/docs/reference/access-authn-authz/authentication/)
+sehingga otentikasi menghasilkan _usernames_ dalam format yang kamu inginkan.
+
+{{< caution >}}
+Awalan `system:` direservasi untuk sistem Kubernetes, jadi kamu harus memastikan
+bahwa kamu tidak memiliki pengguna atau grup dengan nama yang dimulai dengan `system:` secara tidak sengaja.
+Selain awalan khusus ini, sistem otorisasi RBAC tidak memerlukan format apa pun
+untuk nama pengguna.
+{{< /caution >}}
+
+Di Kubernetes, modul otentikasi menyediakan informasi grup.
+Grup, seperti halnya pengguna, direpresentasikan sebagai string, dan string tersebut tidak memiliki format tertentu,
+selain awalan `system:` yang sudah direservasi.
+
+[ServiceAccounts](/docs/tasks/configure-pod-container/configure-service-account/) memiliki nama yang diawali dengan `system:serviceaccount:`, dan menjadi milik grup yang diawali dengan nama `system:serviceaccounts:`.
+
+{{< note >}}
+- `system:serviceaccount:` (tunggal) adalah awalan untuk _service account usernames_.
+- `system:serviceaccounts:` (jamak) adalah awalan untuk _service account_ grup.
+{{< /note >}}
+
+#### Contoh RoleBinding {#role-binding-examples}
```

Comment:
you can remove the custom anchor link here, and refer with `#contoh-rolebinding`
```suggestion
#### Contoh RoleBinding
```